app/src/main/java/com/micutu/locatedriver/Services/SmsSenderService.java

```diff
@@ -310,29 +423,6 @@ public void onDestroy() {
         super.onDestroy();
     }
 
-    public static int getLocationMode(Context context) {
```

Comment:
I didn't remove any of your methods, it's just Git not managing to show it in a pretty diff